py/server.py

```diff
@@ -197,6 +230,12 @@ def on_message(self, message):
                 os.remove(p)
                 broadcast_envs(self)
 
+        if cmd == 'save_layouts':
+            if 'data' in msg:
+                self.layouts = msg.get('data')
+                print(self.layouts)
```

Comment:
This print can be removed. (I presume you were using it for debugging?)

Reply:
Agh thought I got all of the debugs - yeah it made sure the layout was coming through as expected.